gms/gossiper.cc

```diff
@@ -90,7 +88,7 @@ void gossiper::set_cluster_name(sstring name) {
 }
 
 sstring gossiper::get_partitioner_name() {
-    return dht::global_partitioner().name();
+    return _cfg.partitioner();
```

Comment:
This looks weird, is `_cfg.partitioner()` an `sstring` already? In that case it should be called `partitioner_name()`, no?
Or does the partitioner have `operator sstring()`?

Reply:
It is an sstring. I agree the name could be better but it is part of yaml config so changing it is not easy. It would break compatibility with old configs.

Reply:
Ok, no need to change it then.